pkg/lang/ir/v1/julia.go

```diff
@@ -18,44 +18,76 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/cockroachdb/errors"
 	"github.com/moby/buildkit/client/llb"
-	"github.com/sirupsen/logrus"
+)
 
-	"github.com/tensorchord/envd/pkg/types"
+const (
+	julia       = "/opt/julia"                                                                           // Location of downloaded Julia binary and other files
+	juliaBin    = "/opt/julia/bin"                                                                       // Location of Julia executable binary file
+	juliaPkg    = "/opt/julia/user_packages"                                                             // Location of additional packages installed via Julia
+	juliaBinary = "https://julialang-s3.julialang.org/bin/linux/x64/1.8/julia-1.8.3-linux-x86_64.tar.gz" // The official link for downloading Julia environment
```

Comment:
Naming can be clearer. For example:

- juliaRootDir
- juliaBinDir
- juliaPkgDir
- juliaDownloadURL

BTW, there should be a const for Julia version. Julia download URL should be built from the default URL + Julia version.

Reply:
Fixed. Also add other configs